cms/templates/webinar_page.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% load static render_bundle image_version_url wagtailcore_tags wagtailimages_tags wagtailmetadata_tags tz %}
+
+{% block title %}{{ site_name }} | {{ page.title }}{% endblock %}
+
+{% block headercontent %}
+<div id="header"></div>
+{% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+
+  {% if page.banner_image %}
+    <div class="resource-banner" style="background:url('{% image_version_url page.banner_image "original" %}') no-repeat center center;">
```

Comment:
Still some issues with the styling. Please use the styles from the catalog page banner or you can use webinar list page banner image styles. <img width="1006" alt="Screen Shot 2023-07-31 at 4 41 11 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/890ff584-fc71-48cf-a17c-f0f7fd6d06be">